pkgs/os-specific/linux/cpupower/gui.nix

```
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, appstream-glib
+, desktop-file-utils
+, gettext
+, glib
+, gobject-introspection
+, gtk3
+, hicolor-icon-theme
+, libappindicator
+, meson
+, ninja
+, pkg-config
+, python3
+, systemd
+, wrapGAppsHook
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

personal preference to use python3Packages.callPackage
```suggestion
buildPythonApplication rec {
```